db/seeds/source_types.yml

```diff
@@ -123,6 +123,34 @@ ansible-tower:
         :condition:
           :when: endpoint.verify_ssl
           :is: true
+google:
+  :product_name: Google Cloud Engine
```

Comment:
I think we should discuss with UX how to call it, this value is then shown in the ui in multiple places. @beccaglass

Reply:
My preference is 'Google Cloud' -- seems to be the most scalable